packages/tools/package/src/commands/changelog/changelog.utils.ts

```diff
@@ -0,0 +1,118 @@
+import { Package } from '../../models';
+import { AlongWithData, ChangelogEntry } from './changelog.types';
+
+const fs = require('fs');
+const path = require('path');
+const { exec } = require('child_process');
+
+/**
+ * Function to execute shell command and capture output
+ * @param scriptPath - Path to the script which we want to run.
+ * @returns - Promise that resolves once the script is run and returns the output.
+ */
+export function runShellScript(scriptPath: string) {
```

Comment:
This seems generic function. We can check if this is defined somewhere else as well

Reply:
Yes, there is an executor class that is used to run command. Using that class instead on the shell script.